charts/loki-distributed/Chart.yaml

```
@@ -3,7 +3,7 @@ name: loki-distributed
 description: Helm chart for Grafana Loki in microservices mode
```

@JStickler 

Thanks for the contribution. The helm-doc render would README.md.gotmpl -> read this line (`.chart.description`) -> output to README.md, so:

```suggestion
description: "Helm chart for Grafana Loki in microservices mode. This is a community maintained chart."
```